Library/Formula/tesseract.rb

```diff
@@ -65,17 +65,18 @@ class Tesseract < Formula
   end
 
   def install
+    if build.with? "training-tools"
+      icu4c = Formula["icu4c"]
+      ENV.append "CFLAGS", "-I#{icu4c.opt_include}"
+      ENV.append "LDFLAGS", "-L#{icu4c.opt_lib}"
+    end
+
     # explicitly state leptonica header location, as the makefile defaults to /usr/local/include,
     # which doesn't work for non-default homebrew location
     ENV["LIBLEPT_HEADERSDIR"] = HOMEBREW_PREFIX/"include"
 
     ENV.cxx11
 
-    # Fix broken pkg-config file
```

Comment:
it's actually done in code `inreplace "tesseract.pc.in", "@OPENCL_LIB@", "@OPENCL_LDFLAGS@" if build.stable?`